### PR TITLE
Update linux.md

### PR DESCRIPTION
**Pull request by fusion809**
Adding Mageia instructions (based on [#8283](https://github.com/atom/atom/issues/8283)). Adding extra troubleshooting information.

### CODE REVIEW

**Comment by benogle** [comment]
cc @atom/feedback

File: docs/build-instructions/linux.md [hunk 1]
```diff
@@ -128,13 +132,15 @@ See also [#2082](https://github.com/atom/atom/issues/2082).
 ### /usr/bin/env: node: No such file or directory
 
 If you get this notice when attempting to `script/build`, you either do not
-have Node.js installed, or node isn't identified as Node.js on your machine.
-If it's the latter, entering `sudo ln -s /usr/bin/nodejs /usr/bin/node` into
+have Node.js installed, or node is not identified as Node.js on your machine.
+If it is the latter, entering `sudo ln -s /usr/bin/nodejs /usr/bin/node` into
```
Comment (lee-dohm) [inline docs/build-instructions/linux.md]: Is there a reason for eliminating contractions? I feel not using contractions can seem overly formal in something like build instructions.

Comment (maxbrunsfeld) [inline docs/build-instructions/linux.md]: Agreed. @fusion809 In general, it makes a PR easier to merge if you make one focused change. So in this case, could you undo the minor wording changes that you made to the surrounding prose?

**Comment by lee-dohm** [comment]
I'm going through old PRs to see if they can be salvaged or should be closed. Is this something that you still want to finish? Let us know :grinning:

**Comment by damieng** [comment]
No response.

**Pull request closed without merging**